data/forms/add-relationship.tsx

```diff
@@ -1,17 +1,30 @@
 import { FormStep, Option, Options } from 'components/Form/types';
+import { Resident } from 'types';
 
-const formSteps = (relationshipTypeOptions: Options): FormStep[] => {
+const formSteps = (
+  relationshipTypeOptions: Options,
+  secondPerson: Resident
+): FormStep[] => {
   return [
     {
       id: 'add-relationship',
       title: 'Add Relationship',
       components: [
-        <h2
+        <h1
           key="subtitle-details"
           className="govuk-fieldset__legend--l gov-weight-lighter govuk-!-margin-bottom-5"
         >
           Add new relationship
+        </h1>,
+        <h2
```

Comment:
this might be better off as `<p>` tags, because they aren't really headings. you can use class names to make things visually look a particular way

Reply:
Ahhhh, good shout - have updated!